doc/user/content/sql/copy-to.md

```diff
@@ -1,41 +1,164 @@
-_field_ | The name of the option you want to set.
-_val_ | The value for the option.
+### Syntax {#copy-to-stdout-syntax}
 
-### `WITH` options
+{{< diagram "copy-to-stdout.svg" >}}
 
-Name | Value type | Default value | Description
-----------------------------|--------|--------|--------
-`FORMAT` | `TEXT`,`BINARY` | `TEXT` | Sets the output formatting method.
+Field         | Use
+--------------|-----
+_query_       | The [`SELECT`](/sql/select) or [`SUBSCRIBE`](/sql/subscribe) query to output results for.
+_field_       | The name of the option you want to set.
+_val_         | The value for the option.
 
-## Example
+### `WITH` options {#copy-to-stdout-with-options}
 
-### Copying a view
+Name     | Values                 | Default value | Description
+---------|------------------------|---------------|-----------------------------------
+`FORMAT` | `TEXT`,`BINARY`, `CSV` | `TEXT`        | Sets the output formatting method.
+
+### Examples {#copy-to-stdout-examples}
+
+#### Subscribing to a view with binary output
 
 ```sql
-COPY (SELECT * FROM some_view) TO STDOUT;
+COPY (SUBSCRIBE some_view) TO STDOUT WITH (FORMAT binary);
 ```
 
-### Subscribing to a view with binary output
+## Copy to Amazon s3 {#copy-to-s3}
+
+{{< private-preview />}}
+
+Copying results to Amazon s3 is useful to perform tasks like periodic backups
+for auditing, or downstream processing in analytical data warehouses like
+Snowflake, Databricks or BigQuery. For step-by-step instructions, see the
+integration guide for [Amazon s3](/serve-results/s3/).
+
+The `COPY TO` command is _one-shot_: every time you want to export results, you
+must run the command. To automate exporting results on a regular basis, you can
+set up scheduling, for example using a simple `cron`-like service or an
+orchestration platform like Airflow or Dagster.
+
+### Syntax {#copy-to-s3-syntax}
+
+{{< diagram "copy-to-s3.svg" >}}
+
+Field         | Use
+--------------|-----
+_query_       | The [`SELECT`](/sql/select) query to copy results out for.
+_object_name_ | The name of the object to copy results out for.
+**AWS CONNECTION** _connection_name_ | The name of the AWS connection to use in the `COPY TO` command. For details on creating connections, check the [`CREATE CONNECTION`](/sql/create-connection/#aws) documentation page.
+_s3_uri_      | The unique resource identifier (URI) of the Amazon s3 bucket to store the output results in.
+**FORMAT**    | The file format to write.
+_field_       | The name of the option you want to set.
+_val_         | The value for the option.
+
+### `WITH` options {#copy-to-s3-with-options}
+
+Name             | Values          | Default value | Description                       |
+-----------------|-----------------|---------------|-----------------------------------|
+`MAX FILE SIZE`  | `int`           |               | Sets the approximate maximum file size of each file uploaded to the S3 bucket. |
```

Comment:
I actually just realized that this isn't required to be an `int` -- e.g. this also works ` MAX FILE SIZE = '100MB'`